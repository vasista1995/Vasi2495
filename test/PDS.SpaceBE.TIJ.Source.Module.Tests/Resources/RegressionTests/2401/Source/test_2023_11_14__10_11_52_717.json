{
  "PackageFamily": "WDSON",
  "Module": "SP",
  "Package": "WDSON_M-CAN",
  "Segment": "Tijuana",
  "FourDReport": "*",
  "ProcessEquipment": "DBBE0019",
  "ParameterClass": "NON_PPAP",
  "Classification": "SPC-Monitoreo",
  "Operator": "JUAREZJOSEGU",
  "WaferThickness": "SFET3",
  "HexSize": "3",
  "Layout": "MZ",
  "PartNumber": "MA005974507",
  "SolderType": "44-0229",
  "Area": "DirectFet_Diebond",
  "SpaceRawValues": [
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 3096856,
      "Seqnr": 1,
      "Value": 2100.0,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 3096856,
      "Seqnr": 2,
      "Value": 2110.0,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 3096856,
      "Seqnr": 3,
      "Value": 2103.0,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 3096856,
      "Seqnr": 4,
      "Value": 2100.0,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 3096856,
      "Seqnr": 5,
      "Value": 2104.0,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    }
  ],
  "UniqueSourceKey": "TIJ:BE:104OOH5.I01:DirectFet_Diebond:SP:Die_PlacementX:3102:3096856:0",
  "PKey": "TIJ:BE:104OOH5.I01:DirectFet_Diebond:SP:Die_PlacementX:3102:3096856:0",
  "ParameterName": "Die_PlacementX",
  "ParameterUnit": "um",
  "SampleDate": "2023-10-09T19:55:25Z",
  "SampleDateUtc": "2023-10-10T02:55:25Z",
  "SampleSize": 5,
  "Sigma": "4,09878030638384",
  "Mean": "2103,4",
  "Median": "2103",
  "Min": "2100",
  "Max": "2110",
  "Q1": "2100",
  "Q3": "2104",
  "LdsID": "2401",
  "IsCalcParam": 0,
  "MeanCntrlLow": 2080.0,
  "MeanCntrlTarget": 2100.0,
  "MeanCntrlHigh": 2120.0,
  "RangeCntrlLow": 14.13,
  "RangeCntrlTarget": 26.0,
  "RangeCntrlHigh": 42.13,
  "SpecLow": 2000.0,
  "SpecTarget": 2100.0,
  "SpecHigh": 2200.0,
  "CtrlMeanHighEnabled": "Y",
  "CtrlMeanLowEnabled": "Y",
  "CtrlMeanTargetEnabled": "Y",
  "CtrlLowEnabled": "N",
  "CtrlHighEnabled": "N",
  "CtrlRangeLowEnabled": "N",
  "CtrlRangeHighEnabled": "N",
  "CtrlSigmaLowEnabled": "Y",
  "CtrlSigmaCenterEnabled": "Y",
  "CtrlRangeCenterEnabled": "N",
  "CtrlCenterEnabled": "N",
  "CtrlSigmaHighEnabled": "Y",
  "SpecLowEnabled": "Y",
  "SpecHighEnabled": "Y",
  "CreatedDate": "2023-10-09T19:55:25Z",
  "CreatedDateUtc": "2023-10-10T02:55:25Z",
  "UpdatedDate": "2023-10-09T19:55:25Z",
  "UpdatedDateUtc": "2023-10-10T02:55:25Z",
  "RvStoreFlag": "Y",
  "MeasurementEquipment": "MSNE0003",
  "Lot": "104OOH5.I01",
  "ChannelId": "3102",
  "CkcId": "0",
  "ChannelName": "Die_PlacementX/WDSON_M-CAN/MZ/SFET3/ABLESTIK_AAA3131A-IR",
  "SpaceInstanceName": "BE",
  "Sitekey": "TIJ",
  "CalcCreated": "12/05/2022 15:26:28",
  "ILdsID": 0,
  "AcceptFlag": "Y",
  "UpdatedBy": " ",
  "SpecTargetOrigin": "V",
  "ViolCount": 0,
  "ExtMaMV": 2110.32,
  "ExtEwmaMV": 2106.42787474563,
  "ExtMsMV": 0.707106781186548,
  "OutOfOrder": "N",
  "EwmaS": 4.36466340291679,
  "EwmaR": 10.7912243635565,
  "Calc2ID": 90344,
  "LimitEnable": "YYNYNNYNYYYYYYYYYYYNNNYYYYYYYNNNNNNNN",
  "ExtEwmaMVLCL": 0.0,
  "ExtEwmaMVCenter": 0.0,
  "ExtEwmaMVUCL": 0.0,
  "ExtMaMVLCL": 0.0,
  "ExtMaMVCenter": 0.0,
  "ExtMaMVUCL": 0.0,
  "EwmaSLCL": 0.0,
  "EwmaSCenter": 0.0,
  "EwmaSUCL": 0.0,
  "EwmaRLCL": 0.0,
  "EwmaRCenter": 0.0,
  "EwmaRUCL": 0.0,
  "ExtMSMVLCL": 0.0,
  "ExtMSMVCenter": 0.0,
  "ExtMSMVUCL": 0.0,
  "ClOrigin": 0,
  "MVLal": 0.0,
  "MVUal": 0.0,
  "RawLal": 0.0,
  "RawUal": 0.0,
  "SigmaLal": 0.0,
  "SigmaUal": 0.0,
  "RangeLal": 0.0,
  "RangeUal": 0.0,
  "ExtSpecLower": 0.0,
  "ExtSpecUpper": 0.0,
  "ExtSpecLimEnable": "NN",
  "CalculationStrategy": "ND",
  "ChState": "Online"
}