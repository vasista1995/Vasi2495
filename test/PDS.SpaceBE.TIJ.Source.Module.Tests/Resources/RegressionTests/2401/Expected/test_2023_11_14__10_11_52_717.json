{
  "IdSource": "SPACEACT2:TIJ:BE:104OOH5.I01:DirectFet_Diebond:SP:Die_PlacementX:3102:3096856:0",
  "Data_flat_Metadata": {
    "PackageFamily": "WDSON",
    "Package": "WDSON_M-CAN",
    "Segment": "Tijuana",
    "Module": "SP",
    "FourDReport": "*",
    "ParameterClass": "NON_PPAP",
    "ProcessEquipment": "DBBE0019",
    "Classification": "SPC-Monitoreo",
    "Operator": "JUAREZJOSEGU",
    "WaferThickness": "SFET3",
    "HexSize": "3",
    "Layout": "MZ",
    "PartNumber": "MA005974507",
    "SolderType": "44-0229",
    "Area": "DirectFet_Diebond",
    "Lot": "104OOH5.I01",
    "ParameterName": "Die_PlacementX",
    "ParameterUnit": "um",
    "SiteKey": "TIJ",
    "ChannelId": "3102",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-10-10T02:55:25Z",
    "CreatedTimestamp": "2023-10-09T19:55:25Z",
    "UpdatedTimestamp": "2023-10-09T19:55:25Z",
    "UpdatedTimestampUtc": "2023-10-10T02:55:25Z",
    "ExportedTimestampUtc": "2023-10-09T19:55:25Z",
    "ExportedTimestamp": "2023-10-09T19:55:25Z",
    "SampleTimestampUtc": "2023-10-10T02:55:25Z",
    "SampleTimestamp": "2023-10-09T19:55:25Z",
    "SourceDataLevel": "C",
    "Equipment": "MSNE0003",
    "ChannelName": "Die_PlacementX/WDSON_M-CAN/MZ/SFET3/ABLESTIK_AAA3131A-IR",
    "RvStoreFlag": "Y",
    "CalcCreated": "12/05/2022 15:26:28",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 90344,
    "ViolCount": 0,
    "CalculationStrategy": "ND",
    "LdsID": "2401",
    "IsCalcParam": 0,
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "Y",
        "SpecHigh": 2200.0,
        "SpecLow": 2000.0,
        "SpecTarget": 2100.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYNNYNYYYYYYYYYYYNNNYYYYYYYNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "2103,4",
        "Median": "2103",
        "Sigma": "4,09878030638384",
        "Min": "2100",
        "Max": "2110",
        "Q1": "2100",
        "Q3": "2104"
      },
      "ControlLimits": {
        "ctrl_mean_low": 2080.0,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 2100.0,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 2120.0,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "Y",
        "CtrlRangeCenterEnabled": "N",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "Y",
        "CtrlSigmaHighEnabled": "Y",
        "ctrl_range_low": 14.13,
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_target": 26.0,
        "ctrl_range_high": 42.13,
        "CtrlRangeHighEnabled": "N",
        "ExtMaMV": 2110.32,
        "ExtEwmaMV": 2106.42787474563,
        "ExtMsMV": 0.707106781186548,
        "EwmaS": 4.36466340291679,
        "EwmaR": 10.7912243635565,
        "ExtEwmaMVLCL": 0.0,
        "ExtEwmaMVCenter": 0.0,
        "ExtEwmaMVUCL": 0.0,
        "ExtMaMVLCL": 0.0,
        "ExtMaMVCenter": 0.0,
        "ExtMaMVUCL": 0.0,
        "EwmaSLCL": 0.0,
        "EwmaSCenter": 0.0,
        "EwmaSUCL": 0.0,
        "EwmaRLCL": 0.0,
        "EwmaRCenter": 0.0,
        "EwmaRUCL": 0.0,
        "ExtMSMVLCL": 0.0,
        "ExtMSMVCenter": 0.0,
        "ExtMSMVUCL": 0.0,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 0.0,
        "RawUal": 0.0,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 3096856,
        "Seqnr": 1,
        "Value": 2100.0,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096856,
        "Seqnr": 2,
        "Value": 2110.0,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096856,
        "Seqnr": 3,
        "Value": 2103.0,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096856,
        "Seqnr": 4,
        "Value": 2100.0,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096856,
        "Seqnr": 5,
        "Value": 2104.0,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 5
  },
  "ProductionAction": {
    "Id": "SPACEACT1:TIJ:BE:DirectFet_Diebond:SP:Die_PlacementX:3102",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:104OOH5.I01",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "TIJ:BE:104OOH5.I01:DirectFet_Diebond:SP:Die_PlacementX:3102:3096856:0"
  }
}